Tests/Features/Store/NotificationStoreCombineTests.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  NotificationStoreCombineTests.swift
+//  MagicBell
+//
+//  Created by Javi on 27/12/21.
+//
+
+@testable import MagicBell
+@testable import Harmony
+import struct MagicBell.Notification
+import XCTest
+import Nimble
+import Combine
+
+class NotificationStoreCombineTests: XCTestCase {
```

Comment:
@jfilowk missing tests for `fetch` and `refresh` methods used with Futures. Also, all other tests should be using the Combine based (Futures) fetch and refresh (instead of the completion block based)

Reply:
Also, missing tests on `NotificationStorePublishers` class.